### ISSUE TITLE
[Suggestion] Stone Drawers

### ISSUE THREAD

**Issue by Xiaminou** [jaquadro/StorageDrawers#589]
I have always wanted to have a Basic Drawer that looks similar to the Compacting Drawer, but I've never been able to replicate it with the Framed Drawers.
The Stone Drawers would be the same as Basic Drawers (single, double and quadruple) but made with stone and that would look the same as a Compacting Drawer.
It would also help solve compatibility issues that come with using mods that add different types of woods. When playing with many mods sometimes you just don't have access to vanilla woods. And while I know that there is a mod for this, there will always be some missing.

TL;DR: Basic Drawers made out of stone that look similar to Compacting Drawers.

**Comment by hron84** [comment]
Also, I'm looking for a frameable compacting drawer :stuck_out_tongue:

**Comment by jaquadro** [comment]
If you've got a mod providing a full block stone slab, then you can probably get something close by combining that with a dark block like black powdered concrete or multi-compressed cobble.

(framed compacting is still on the list)

**Comment by hron84** [comment]
> (framed compacting is still on the list)

Thanks!

**Comment by Frontrider** [comment]
I placed the controller slave on the outside of the drawer (yes it works), so the sides match.

**Comment by jaquadro** [comment]
Good tip (if somewhat expensive 💎 )

**Comment by Frontrider** [comment]
Yeah, I only use it sparingly.

**Comment by hron84** [comment]
@jaquadro could you just provide a decoration block (trim?) that uses side texture of the compacting/controller/slave textures? Then it could be used for framing. Not as easy as the opening question but it could have more use.

**Comment by hron84** [comment]
@Frontrider also it could be darker but still used if you can grab some crucible-like thing and use it for framing. Maybe EIO decoration blocks can also used.

**Comment by Frontrider** [comment]
@hron84 I used black concrete powder, and its actually good enough.

You have to experiment with that, your pack might have better options.

**Comment by Xiaminou** [comment]
![2018-04-20_13 18 03](https://user-images.githubusercontent.com/8670392/39048519-0c7e9180-449e-11e8-9092-9675f1dbfcad.png)
![2018-04-20_13 18 05](https://user-images.githubusercontent.com/8670392/39048520-0c9a5d7a-449e-11e8-8cbe-95b10438ff8d.png)
![2018-04-20_13 18 09](https://user-images.githubusercontent.com/8670392/39048521-0cb38660-449e-11e8-970d-d4d9735f4207.png)
![2018-04-20_13 18 49](https://user-images.githubusercontent.com/8670392/39048522-0ccb2310-449e-11e8-9d0d-d44c77cb979d.png)

Here is my attempt, it uses Polished Stone from Quark and Asphalt Concrete from Immersive Petroleum.

**Comment by hron84** [comment]
@Xiaminou Perfect!
